site/content/3.10/get-started/how-to-interact-with-arangodb.md

```diff
@@ -2,9 +2,12 @@
 title: How to Interact With ArangoDB
 menuTitle: How to Interact with ArangoDB
 weight: 50
-description: ''
+description: >-
+    Learn about the different ways you can use and communicate with ArangoDB
```

Comment:
Indentation should always be 2 spaces in YAML
```suggestion
  Learn about the different ways you can use and communicate with ArangoDB
```

Reply:
Fixed all occurrences.